LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
This license must appear as a header, together with copyright notice, on every source file of the project.

Reply:
See ZIO for an example. ^^^